Kconfig

```diff
@@ -109,6 +109,14 @@ config AOS_SOCKET_SERVER_PORT
 	depends on NATIVE_APPLICATION
 	default 30001
 
+config AOS_PBHANDLER_THREAD_STACK_SIZE
+	int "Aos PB handler stack size"
+	default 16384
+
+config AOS_LAUNCHER_THREAD_STACK_SIZE
+	int "Aos launcher stack size"
+	default 16384
```

Comment:
Do we need different constants for launcher & pb threads?
Also it might be better to set a stack size in pages & have different value for it. Like: 
default 8 * PAGE_SIZE

Reply:
Yes, we need, for better stack adjustment . We don't know PAGE_SIZE at compile time. And it is differ on different platform.